LDAR_Sim/src/file_processing/input_processing/input_manager.py

```diff
@@ -273,7 +273,10 @@ def parse_parameters(self, new_parameters_list: list[dict]):
             # methods pre-specified
             for midx, method in program[pc.Levels.METHOD].items():
                 if ParameterProcessingConst.DEFAULT_PARAM_STRING not in method:
-                    def_file = df.METH_DEF_FILE
+                    if method[pc.Method_Params.DEPLOYMENT_TYPE] == pc.Deployment_Types.MOBILE:
+                        def_file = df.METH_MOBILE_DEF_FILE
+                    else:
```

Comment:
Can we make this an else if and an else with an error for invalid deployment type